apps/scraper/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
+from .models import Site
+
+
+class ChoseSiteForm(forms.Form):
+    for site in range(len(Site.objects.all())):
```

Comment:
You can use values() method of Django QuerySet to retrieve id's of objects:
https://docs.djangoproject.com/en/2.2/ref/models/querysets/#values
and you hit database only once. In your implementation you hit database 2*len(queryset) times.